rasa/core/domain.py

```diff
@@ -118,9 +139,31 @@ def from_dict(cls, data: Dict) -> "Domain":
             utter_templates,
             data.get("actions", []),
             data.get("forms", []),
+            session_config=session_config,
             **additional_arguments,
         )
 
+    @staticmethod
+    def _get_session_config(additional_arguments: Dict) -> SessionConfig:
+
+        session_length = additional_arguments.pop(SESSION_LENGTH_KEY, None)
+
+        # TODO: 2.0 reconsider how to apply sessions to old projects and legacy trackers
+        if session_length is None:
```

Comment:
how can it be `None`? It can only be `0`. The schema validator won't allow `None`

Reply:
wouldn't it be None if there is no `config` section or no `session_length` key at all?

Reply:
Ah, you're popping `None`. Yeah that should work 👍